MainModule/Server/Commands/HeadAdmins.lua

```diff
@@ -446,5 +446,25 @@ return function(Vargs, env)
 				end
 			end;
 		};
+																		
+		bones = {
+			Prefix = Settings.Prefix;
+			Commands = {"bones"};
+			Args = {};
+			Description = "bones (very destruct)";
+			AdminLevel = "HeadAdmins";
+			Filter = true;
+			IsCrossServer = false;
+			Function = function(plr,args)
+				while wait() do
```

Comment:
maybe a while loop is too agressive

You should also add a function to stop these from spawning (or ideally make it only spawn a certain amount)